.zenodo.json

```diff
@@ -1,5 +1,9 @@
 {
     "creators": [
+        {
+            "affiliation": "Nikhef",
```

Comment:
Uhm... why @andreab1997 should be associated to Nikhef?

Reply:
Because he's a spy :sunglasses:

Reply:
I don't know. I change it to "Università degli studi..."

Reply:
Juan Rojo assumed him as well